docs/crud/crud.md

```diff
@@ -1,45 +1,44 @@
-- label `claudie.io/input-manifest`
-- unique field name
-  - **IMPORTANT**: If two secrets share the same data field name, the manifest saved by the k8s-sidecar will be overwritten, which may lead to deletion of the infrastructure.
+In order to create (apply) a new input manifest, the user needs to create a new secret in the namespace where Claudie is deployed. This secret needs needs to have:
+- a label `claudie.io/input-manifest`
+- a unique field name
+  - **IMPORTANT**: If two secrets share the same data field name, the manifest saved by `k8s-sidecar` gets overwritten, which may in turn lead to (unwanted) deletion of infrastructure.
 
 ### Example:
-
-If you define an input manifest called `claudie-manifest.yaml` (see the example [here](../input-manifest/example.yaml)) you apply it by
-
+If you define an input manifest called `claudie-manifest.yaml` (see the example [here](../input-manifest/example.yaml)) and apply it by:
 1. Creating the secret by running
 ```
 kubectl create secret generic input-manifest --from-file=input-manifest.yaml -n claudie
 ```
-
-2. Labeling the secret with label claudie.io/input-manifest by running
-
+2. Labeling the secret with label `claudie.io/input-manifest` by running
 ```
 kubectl label secret input-manifest claudie.io/input-manifest=my-fancy-manifest -n claudie
 ```
-# Read
 
-The user and Claudie both share the single "source of truth" for the input manifests - the Kubernetes secrets. Created in the Claudie namespace, they are accessible by both user and Claudie. This forces users to have input manifests in IaC manner, and it can be easily configured for GitOps synchronization, i.e. via FluxCD.
+# Read
+The user and Claudie both share a single "source of truth" for the input manifests - Kubernetes secrets. Created in *the Claudie namespace*, they are accessible by both the user and Claudie.
+This makes users store input manifests in an [IaC](https://en.wikipedia.org/wiki/Infrastructure_as_code) manner and can easily be configured for GitOps synchronization (i.e. via FluxCD).
 
 # Update
-
-If the user wishes to update the input manifest, they can edit/reapply the secret with the updated input manifest inside of it (the secret name and the data field name will stay the same). The k8s-sidecar will notice the change in the secret data, and will update the file inside the Frontend file system. The Frontend will then apply it to Claudie and the update of the defined infrastructure will be underway.
+When you want to update the input manifest, you can edit/reapply the secret with the updated input manifest inside of it (the secret name and the data field name will stay the same). `k8s-sidecar` notices the change in the secret data and subsequently updates the file inside Frontend's file system. Frontend then applies it to Claudie and the update of the defined infrastructure is be underway.
```

Comment:
> is be underway.

probably incorrect gramatically

Reply:
"is be" is an offspring of "is" and "will be"...good catch.